test/views/CompletedReturnsViewSpec.scala

```diff
@@ -53,7 +53,7 @@ class CompletedReturnsViewSpec extends ViewBaseSpec {
 
   "Rendering the VAT Returns page with no returns for the selected year of 2018" should {
 
-    lazy val view = views.html.returns.completedReturns(VatReturnsViewModel(returnYears, 2018, Seq()))
+    lazy val view = views.html.returns.completedReturns(VatReturnsViewModel(returnYears, 2018, Seq(), hasNonMtdVat = false))
```

Comment:
I think there should be a test up here to check the 'previous returns' tab shouldn't be rendered as the test data doesn't meet either criteria